client/src/client.ts

```diff
@@ -1146,7 +1146,7 @@ export class LanguageClient {
 		this.refreshTrace(connection, false);
 		let initOption = this._clientOptions.initializationOptions;
 		let initParams: InitializeParams = {
-			processId: process.pid,
+			processId: typeof process !== 'undefined' ? process.pid : 0,
```

Comment:
Should be `null`, not `0`
https://github.com/sourcegraph/language-server-protocol/blob/master/protocol.md#initialize